samcli/lib/providers/sam_function_provider.py

```diff
@@ -87,7 +86,7 @@ def _deprecate_notification(self, runtime):
             )
             LOG.warning(self._colored.yellow(message))
 
-    def get_all(self):
+    def get_all(self) -> Generator[Function, None, None]:
```

Comment:
Functions that only `yield` can be annotated as `Iterator[Function]`: https://docs.python.org/3/library/typing.html#typing.Generator

Reply:
good one thx!